### PR TITLE
Add cache to isJwtTokenValid and create turso.Tokens client

### PR DESCRIPTION
**Pull request by athoscouto**

### CODE REVIEW

File: internal/turso/turso.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ type Client struct {
 	Feedback      *FeedbackClient
 	Organizations *OrganizationsClient
 	Locations     *LocationsClient
+	Tokens        *TokensClient
```
Comment (haaawk) [inline internal/turso/turso.go:28]: Where is TokensClient defined?

File: internal/settings/cache.go [hunk 2]
```diff
@@ -99,3 +106,19 @@ func (s *Settings) LocationsCache() (map[string]string, string) {
...
+	setCacheWithExp(key, exp, true)
+}
+
+func (s *Settings) TokenValidCache(token string) bool {
+	key := TOKEN_VALID_CACHE_KEY_PREFIX + strings.ReplaceAll(token, ".", "_")
+	ok, err := getCache[bool](key)
+	if err != nil {
+		return false
+	}
+	return ok
```
Comment (haaawk) [inline internal/settings/cache.go:123]: Ultra nit: This is just `return err != nil && ok`

File: internal/settings/cache.go [hunk 3]
```diff
@@ -23,10 +24,16 @@ func cacheKey(key string) string {
 }
 
 func setCache[T any](key string, ttl int64, value T) error {
-	entry := Entry[T]{Data: value}
+	exp := int64(0)
 	if ttl > 0 {
-		entry.Expiration = time.Now().Unix() + ttl
+		exp = time.Now().Unix() + ttl
 	}
```
Comment (haaawk) [inline internal/settings/cache.go:30]: Why not just `exp := time.Now().Unix() + ttl`. That would be preserving the previous behaviour. Setting exp to 0 for ttl = 0 changes how this case is handled in getCache.

File: internal/turso/turso.go (hunk 1 above)
Comment (athoscouto) [reply internal/turso/turso.go:28]: On `internal/turso/token.go`. But it wasn't there. Rebase hell made it end up in the location PR by accident.

File: internal/settings/cache.go (hunk 2 above)
Comment (athoscouto) [reply internal/settings/cache.go:123]: Will do! But you almost got me there 😬

File: internal/settings/cache.go (hunk 3 above)
Comment (athoscouto) [reply internal/settings/cache.go:30]: I wanted non-expiring entries as well.  I don't remember why I wanted them.
And we don't have any, so I'll remove this.

Comment (haaawk) [reply internal/settings/cache.go]: Currently entry.Expiration == 0 means returning nil in getCache.

**Review by haaawk — APPROVED** [review]

**Pull request merged**